docs/tabular_buckets.md

```diff
@@ -0,0 +1,86 @@
+# Tabular Bucket Types
+## Abstract
+Riak TS is built upon Riak KV which is built upon Riak Core in so much as
+Riak Core was factored out of a monolithic Riak KV and in so much as Riak TS
+utilizes several function call paths of Riak KV but fundamentally differs
+as neither the table nor a record within a table is a Riak Object in the sense
+that consumers as well as developers of Riak KV have committed to memory, the
+rules to reason about to guarantee eventual consistency in an AP system. To
+create a coherent understanding of Riak TS Table and Record entities, we will
+explore the operations that act upon these entities with specific callouts to
+differences between the TS entities and Riak KV ~entities Bucket Type, Bucket,
```

Comment:
Afaik ~ in documentation means "not quite" or approximately, which in this case should be understood as "not quite" b/c neither a Riak KV Bucket Type nor a Bucket is an entity.